docs_src/callbacks.csv_logger.ipynb

```diff
@@ -32,7 +32,7 @@
       "text/markdown": [
```

Comment:
<p>There is a weird warning here, can you get rid of it?</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1833/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yeap, let me look into that, thanks :)</p>

Reply:
<p>Issue seems to be addressed here https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation#how-to-disable-the-warning. But it's weird if I am the only one getting it or is it related to 		`pyyaml` version?</p>

Reply:
I don't have it, personally.